_posts/2011-01-02-app-self-service.md

```diff
@@ -0,0 +1,61 @@
+
+Below is an illustration of the App Self-Service flow:
+
+#### 1. Switching from Developer Portal to Mission Control
+Navigate to the Account section
+<img class="shadow image" src="../assets/app_self_service_1.png" alt="AppSelfService_Screenshot1" style="border:20px;margin:20px">
+
+Click on `Mission Control` to switch views. <br/>
+If you do not see the `Mission Control` link, please contact `devsupport@poynt.co` for assistance.
+<img class="shadow image" src="../assets/app_self_service_2.png" alt="AppSelfService_Screenshot2" style="border:20px;margin:20px">
+
+#### 2. Navigate to Terminal Groups
+Go to Terminal->Groups from the menu and select the group. 
+<div class="note"><span style="font-weight:bold">IMPORTANT!</span> When a new apk is uploaded a terminal group is automatically created for beta testing.
+The group is used to track terminals that have installed the app during beta testing phase. When an app is installed/uninstalled on a merchant terminal, serial number are added/removed from the beta testing terminal group </div>
+<img class="shadow image" src="../assets/app_self_service_3.png" alt="AppSelfService_Screenshot3" style="border:20px;margin:20px">
+<img class="shadow image" src="../assets/app_self_service_6.png" alt="AppSelfService_Screenshot6" style="border:20px;margin:20px">
+<!-- <img class="shadow image" src="../assets/app_self_service_4.png" alt="AppSelfService_Screenshot4" style="border:20px;margin:20px"> -->
+
+#### 3. Create Jobs 
+You may Install and Uninstall apps on terminals
+<!-- <img class="shadow image" src="../assets/app_self_service_5.png" alt="AppSelfService_Screenshot5" style="border:20px;margin:20px"> -->
+<img class="shadow image" src="../assets/app_self_service_6_1.png" alt="AppSelfService_Screenshot6_1" style="border:20px;margin:20px">
+<img class="shadow image" src="../assets/app_self_service_7.png" alt="AppSelfService_Screenshot7" style="border:20px;margin:20px">
+<!-- <img class="shadow image" src="../assets/app_self_service_8.png" alt="AppSelfService_Screenshot8" style="border:20px;margin:20px"> -->
+<img class="shadow image" src="../assets/app_self_service_9.png" alt="AppSelfService_Screenshot9" style="border:20px;margin:20px">
+
+To create a job on specific terminals, use the terminal search box to filter on the required terminals(as comma-separated values).
+<img class="shadow image" src="../assets/app_self_service_10.png" alt="AppSelfService_Screenshot10" style="border:20px;margin:20px">
+
+#### 4. Swithing back to Developer Portal
```

Comment:
Also need to mention that the user needs to have specific permissions (add permissions view screenshot) to be able to schedule jobs.